### PR TITLE
backport: AUT-1185 pcre limit handler

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
- Backport: [AUT-1185](https://oat-sa.atlassian.net/browse/AUT-1185) pcre limit handler #1913

### CODE REVIEW

File: model/qti/container/Container.php [hunk 1]
```diff
@@ -222,6 +223,19 @@ public function fixNonvoidTags($html)
             // correctly closed element
             return trim(mb_substr($matches[0], 0, -2), 'UTF-8') . '></' . $matches[2] . '>';
         }, $html);
+
+        $pregLastError = preg_last_error();
```
Comment (andreluizmachado) [inline model/qti/container/Container.php:227]: Why did you remove the private method?

File: model/qti/container/Container.php [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 namespace oat\taoQtiItem\model\qti\container;
 
+use Monolog\Logger;
```
Comment (pnal) [inline model/qti/container/Container.php:25]: did not notice that in the initial PR, 
asking now, what is the purpose of adding it here?

File: model/qti/container/Container.php (hunk 1 above)
Comment (bartlomiejmarszal) [reply model/qti/container/Container.php:227]: Extracting this method with argument type hint cause issues because `$content` can be string, array or null. Extracting this business logic to separate method was useless and less readable.

**Review by andreluizmachado — APPROVED** [review]
